src/commcare_cloud/ansible/deploy_db.yml

```diff
@@ -32,15 +32,30 @@
   hosts:
     - postgresql
   vars:
-    - username: postgres
+    - userFrom: postgres
     - hostTo: "{{ hot_standby_server }}"
+    - userTo: postgres
     - state: 'present'
   tasks:
     - import_tasks: roles/setup_auth_keys.yml
       when: hot_standby_server is defined
   tags:
     - pg_standby
 
+- name: Setup auth for barman
```

Comment:
I think this should be moved to the barman role to keep it together with the other auth setup

Reply:
I kept it here to club it with other auth such as pg_standby etc.

Reply:
I still think this should be moved since it's unrelated to the other auth setup and since you already have a similar step in the barman role to do the reverse auth.